pydicom/tests/test_filewriter.py

```diff
@@ -297,8 +297,10 @@ def test_write_OD_explicit_little(self):
         # VR (OD): \x4f\x44
         # Reserved: \x00\x00
         # Length (16): \x10\x00\x00\x00
-        #             | Tag          | VR    | Rsrvd |   Length      |    Value ->
-        ref_bytes = b'\x70\x00\x0d\x15\x4f\x44\x00\x00\x10\x00\x00\x00' + bytestring
+        #             | Tag          | VR    |
+        ref_bytes = b'\x70\x00\x0d\x15\x4f\x44' \
+                    b'\x00\x00\x10\x00\x00\x00' + bytestring
+        #             |Rsrvd |   Length      |    Value ->
         self.assertEqual(encoded_elem, ref_bytes)
```

Comment:
I think the way [I did it](https://github.com/pydicom/pydicom/pull/443/commits/aa7782fc4714d0b9e5dd11f558f4494f64f79de6) is a little clearer (avoids the excessive comment):
``` python
        ref_bytes = (b'\x70\x00\x0d\x15'  # tag
                     b'OD'  # VR
                     b'\x00\x00'  # reserved
                     b'\x10\x00\x00\x00'  # length (16)
                     + bytestring)
```
but that's a matter of taste - I'm also ok with this one.